### PR TITLE
[docs][control-plane-manager] Add instruction on how to add master nodes in a cloud cluster

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Added instruction to control-plane-manager FAQ on how to add master nodes in a cloud cluster

Ref: #333

## Changelog entries
```changes
module: documentation
type: fix
```

### CODE REVIEW

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -4,7 +4,58 @@ title: "Управление control plane: FAQ"
 
 ## Как добавить master-узел?
 
-Поставить на узле кластер label `node-role.kubernetes.io/master: ""`, все остальное произойдет полностью автоматически.
+### Cтатичный или гибридный кластер
+Чтобы добавить статичный master-узел в кластер, выполните следующие действия:
+- [Добавьте](https://deckhouse.io/ru/documentation/v1/modules/040-node-manager/faq.html#как-автоматически-добавить-статичный-узел-в-кластер) в кластер статичный узел. `NodeGoup` может быть любая, с параметром `NodeType` — `Static`.
```
Comment (nabokihms) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:9]: Oh, no. To add master nodes to a static cluster, you need to use the manual-boostrap-for-master script, as per other nodes of other node groups. Thus, here you only need to write something like 
> Для добавления мастер ноды в статический кластер, воспользуйтесь инструкцией[ссылка] для добавления статической ноды. Компоненты control-plane будут добавлены на ноду автоматически.

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 2]
```diff
@@ -4,7 +4,58 @@ title: "Управление control plane: FAQ"
...
+
+Чтобы добавить в облачный кластер один или несколько master-узлов, выполните следующие действия:
+- Определите версию и редакцию Deckhouse используемую в кластере. Для этого, выполните на master-узле, или на хосте с настроенным доступом kubectl в кластер:
+  ```shell
+  kubectl -n d8-system get deployment deckhouse \
+  -o jsonpath='versio-{.metadata.annotations.core\.deckhouse\.io\/version}, revision-{.metadata.annotations.core\.deckhouse\.io\/edition}' \
+  | tr '[:upper:]' '[:lower:]'
+  ```
+    
+  Пример вывода команды:
```
Comment (nabokihms) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:26]: This block is excessive. You can move the example to the upper block and delete this one.

**Review by nabokihms — CHANGES_REQUESTED** [review]
I added a couple of suggestions, but overall it looks good.

File: modules/040-control-plane-manager/docs/FAQ_RU.md (hunk 1 above)
Comment (z9r5) [reply modules/040-control-plane-manager/docs/FAQ_RU.md]: Done

File: modules/040-control-plane-manager/docs/FAQ_RU.md (hunk 2 above)
Comment (z9r5) [reply modules/040-control-plane-manager/docs/FAQ_RU.md]: Done

**Comment by deckhouse-BOaTswain** [comment]
Run workflow "deploy/web/stage"...

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 3]
```diff
@@ -3,8 +3,45 @@ title: "Managing control plane: FAQ"
...
 
-All you need to do is to attach the `node-role.kubernetes.io/master: ""` label to a cluster node (all other actions are performed automatically).
+### Cloud cluster
+> Make sure you have all the necessary quota limits, before adding nodes.
+
+To add one or more master nodes to a cloud cluster, follow the following steps:
+- Determine the Deckhouse version and edition used in the cluster by running the following command on the master node or a host with configured kubectl access to the cluster: 
+  ```shell
+  kubectl -n d8-system get deployment deckhouse \
+  -o jsonpath='versio-{.metadata.annotations.core\.deckhouse\.io\/version}, edition-{.metadata.annotations.core\.deckhouse\.io\/edition}' \
```
Comment (nabokihms) [inline modules/040-control-plane-manager/docs/FAQ.md:16]:
```suggestion
  -o jsonpath='version-{.metadata.annotations.core\.deckhouse\.io\/version}, edition-{.metadata.annotations.core\.deckhouse\.io\/edition}' \
```

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 4]
```diff
@@ -3,8 +3,45 @@ title: "Managing control plane: FAQ"
...
 
 ## How do I add a master node?
+### Static or hybrid cluster
+Adding a master node to a static or hybrid cluster has no difference from adding a regular node to a cluster. To do this, use the corresponding [instruction](https://deckhouse.io/en/documentation/v1/modules/040-node-manager/faq.html#how-do-i-automatically-add-a-static-node-to-a-cluster). All the necessary actions to configure a cluster control plane components on the new master nodes are performed automatically. Wait until the master nodes appear in Ready status.
 
-All you need to do is to attach the `node-role.kubernetes.io/master: ""` label to a cluster node (all other actions are performed automatically).
+### Cloud cluster
+> Make sure you have all the necessary quota limits, before adding nodes.
+
+To add one or more master nodes to a cloud cluster, follow the following steps:
```
Comment (nabokihms) [inline modules/040-control-plane-manager/docs/FAQ.md:12]: > follow the following

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 5]
```diff
@@ -3,8 +3,45 @@ title: "Managing control plane: FAQ"
...
+  ```shell
+  dhctl config edit provider-cluster-configuration --ssh-agent-private-keys=/tmp/.ssh/<SSH_KEY_FILENAME> --ssh-user=<USERNAME> \
+  --ssh-host <SSH_HOST>
+  ```
+- Specify the required number of master node replicas in the `masterNodeGroup.replicas` field and save changes.
+- Start scaling process by running the following command (specify the appropriate cluster access parameters, as in the previous step):
+  ```shell
+  dhctl converge --ssh-agent-private-keys=/tmp/.ssh/<SSH_KEY_FILENAME> --ssh-user=<USERNAME> --ssh-host <SSH_HOST>
+  ```
+- Answer `Yes` to the question `Do you want to CHANGE objects state in the cloud?` on the `terraform apply` step.
```
Comment (nabokihms) [inline modules/040-control-plane-manager/docs/FAQ.md:42]: It seems we do not need to mention the `terraform apply` stage here. 
```
Answer `yes` to the question `Do you want to CHANGE objects state in the cloud?`
```

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 6]
```diff
@@ -3,8 +3,45 @@ title: "Managing control plane: FAQ"
...
+  -o jsonpath='versio-{.metadata.annotations.core\.deckhouse\.io\/version}, edition-{.metadata.annotations.core\.deckhouse\.io\/edition}' \
+  | tr '[:upper:]' '[:lower:]'
+  ```
+- Run the corresponding version and edition of the Deckhouse installer:
+  ```shell
+  docker run --pull=always -it -v "$HOME/.ssh/:/tmp/.ssh/" \
+  registry.deckhouse.io/deckhouse/<DECKHOUSE_EDITION>/install:<DECKHOUSE_VERSION> bash
+  ```
+
+  For example, if the Deckhouse version in the cluster is `v1.28.0` and the Deckhouse edition is `ee`, the command to run the installer is:
```
Comment (nabokihms) [inline modules/040-control-plane-manager/docs/FAQ.md:25]:
```suggestion
  For example, if the Deckhouse version in the cluster is `v1.28.0` and the Deckhouse edition is `ee`, the command to run the installer will be:
```

**Review by nabokihms — APPROVED** [review]
Overall, it looks good.
@konstantin-axenov, do you mind also participating in this review?

File: modules/040-control-plane-manager/docs/FAQ.md (hunk 4 above)
Comment (z9r5) [reply modules/040-control-plane-manager/docs/FAQ.md]: Fixed. (used "follow these steps")

File: modules/040-control-plane-manager/docs/FAQ.md (hunk 5 above)
Comment (z9r5) [reply modules/040-control-plane-manager/docs/FAQ.md]: Fixed

**Comment by deckhouse-BOaTswain** [comment]
Run workflow "deploy/web/test"...

**Comment by deckhouse-BOaTswain** [comment]
Run workflow "deploy/web/test"...

**Comment by deckhouse-BOaTswain** [comment]
Run workflow "deploy/web/test"...

**Comment by z9r5** [comment]
Cherry-picked into release-1.29

**Pull request merged**